src/contracts/pods/EigenPod.sol

```diff
@@ -59,43 +55,21 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
     /// @notice The single EigenPodManager for EigenLayer
     IEigenPodManager public immutable eigenPodManager;
 
-    ///@notice The maximum amount of ETH, in gwei, a validator can have restaked in the eigenlayer
-    uint64 public immutable MAX_RESTAKED_BALANCE_GWEI_PER_VALIDATOR;
-
     /// @notice This is the genesis time of the beacon state, to help us calculate conversions between slot and timestamp
     uint64 public immutable GENESIS_TIME;
 
-    // STORAGE VARIABLES
-    /// @notice The owner of this EigenPod
-    address public podOwner;
-
-    /**
-     * @notice The latest timestamp at which the pod owner withdrew the balance of the pod, via calling `withdrawBeforeRestaking`.
-     * @dev This variable is only updated when the `withdrawBeforeRestaking` function is called, which can only occur before `hasRestaked` is set to true for this pod.
-     * Proofs for this pod are only valid against Beacon Chain state roots corresponding to timestamps after the stored `mostRecentWithdrawalTimestamp`.
-     */
-    uint64 public mostRecentWithdrawalTimestamp;
-
-    /// @notice the amount of execution layer ETH in this contract that is staked in EigenLayer (i.e. withdrawn from the Beacon Chain but not from EigenLayer),
-    uint64 public withdrawableRestakedExecutionLayerGwei;
-
-    /// @notice an indicator of whether or not the podOwner has ever "fully restaked" by successfully calling `verifyCorrectWithdrawalCredentials`.
-    bool public hasRestaked;
-
-    /// @notice This is a mapping of validatorPubkeyHash to timestamp to whether or not they have proven a withdrawal for that timestamp
-    mapping(bytes32 => mapping(uint64 => bool)) public provenWithdrawal;
+    /// @notice If a validator is slashed on the beacon chain and their balance has not been checkpointed
+    /// within `TIME_TILL_STALE_BALANCE` of the current block, they are eligible to be marked "stale"
+    /// via `verifyStaleBalance`.
+    uint256 internal constant TIME_TILL_STALE_BALANCE = 2 weeks;
```

Comment:
so this means people would have 2 weeks to update the balance of a validator that gets slashed on the beacon chain?
we should discuss the right value for this, but am fine leaving that as a detail until later.

Reply:
If their last proof for this validator was 2 weeks ago and the validator gets slashed, they only have 6 hours.

OTOH if they proved the validator like last block and then they got slashed, they have 2 weeks!